src/survey-public/components/core/form/SurveyLabeledRadiobuttonGroup.tsx

```diff
@@ -1,3 +1,5 @@
+import clsx from "clsx"
```

Comment:
```suggestion
import { clsx } from "clsx"
```

Der named import ist der "bessere" und die Library migriert langsam dorthin https://github.com/lukeed/clsx/releases (soweit ich das verstehe); ich versuche es einheitlich zu halten auf dem "named import" (das ist häufig der 2. in der identischen liste in VS Code)